tests/migration/migration.go

```diff
@@ -1848,7 +1823,7 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 			})
 		})
 
-		Context("live migration cancelation", func() {
+		Context("live migration cancellation", func() {
```

Comment:
I recently almost fixed a "canceled" the same way, but decided to google it and found out both spellings are correct :)

Reply:
I just trust in Goland spell check xD